assets/src/blocks/CarouselHeader/CarouselControls.js

```diff
@@ -20,21 +20,17 @@ export const CarouselControls = ({
     {/* Indicators */}
     <div className='carousel-indicators-wrapper'>
       <div className='container'>
-        <div className='row'>
```

Comment:
These classes `row` and `col` didn't seem to add anything to the layout, so I removed them